backend_new/prisma/seed-helpers/application-preference-factory.ts

```diff
@@ -0,0 +1,78 @@
+import { MultiselectQuestions, Prisma } from '@prisma/client';
+import { randomNoun } from './word-generator';
+import { randomBoolean } from './boolean-generator';
+import { InputType } from '../../src/enums/shared/input-type-enum';
+
+// TODO map to actual multiselect questions
```

Comment:
Is this todo/commented out code still needed?

Reply:
nope, removed!